lib/web/css/source/lib/_resets.less

```diff
@@ -55,7 +55,7 @@
     object,
     video,
     embed {
-        max-height: 100%;
+        height: auto;
```

Comment:
As mentioned in the comment, this change was a part of https://github.com/magento/magento2/commit/d0ad63cca0b8e29b86b29c72608d034658d244a5 and reverting it would most probably break something else.

My suggestion would be to split `img` from this bunch and also remove the style that was added for logo if we can have it globally.

Reply:
Hello @krzksz,
Thanks for your suggestions. I have review your comment and agree with the same. As per your suggestion, I have added sperate properties for `img`, please review